sdks/go/pkg/beam/core/runtime/xlangx/expansionx/process_test.go

```diff
@@ -44,19 +57,38 @@ func TestNewExpansionServiceRunner(t *testing.T) {
 	}
 }
 
+func TestGetPort(t *testing.T) {
+	testPort := "8097"
+	serviceRunner, err := NewExpansionServiceRunner("", testPort)
+	if err != nil {
+		t.Fatalf("NewExpansionServiceRunner failed, got %v", err)
+	}
+	observedPort := serviceRunner.GetPort()
+	expPort := "localhost:" + testPort
+	if observedPort != expPort {
+		t.Errorf("GetPort() returned mismatched value: wanted %v, got %v", observedPort, expPort)
```

Comment:
Nit: got & want for observedPort and expPort respectively are sufficient. There's nothing to confuse them with, and using the got + want convention will make it easier on other readers for consistency.

Reply:
Fixed, that makes sense